### DISCUSSION TITLE
星铁美照，每日一发

### DISCUSSION THREAD

**Discussion by XYdtman** [share121/inter-knot#3333] [其他]
![StarRail_Image_1723624542](https://github.com/user-attachments/assets/343c0e6a-5dba-4847-abc4-a16f8ca6fbc0)
角色：三月七--存护
角色获取方式：御三家
地点：贝洛伯格花店
设置：除了特效全拉最高
n卡滤镜：关
游戏内滤镜：少女78
待机动作：2（1也不错）

希望有人能教我用ps调光线<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >

**Comment by 21Seven** [comment]
好！
